spectrum/checks.py

```diff
@@ -503,8 +503,17 @@ def generic(self, path):
         LOGGER.info("Loading %s", url)
         response = requests.get(url)
         _assert_status_code(response, 200, url)
+        _assert_all_resources_of_page_load(response.content, self._host)
         return response.content
 
+    def listing(self, path):
+        body = self.generic(path)
+        soup = BeautifulSoup(body, "html.parser")
+        teaser_a_tags = soup.select("div.teaser h4 a")
```

Comment:
Would be nice to make it more precise. Can you confirm:
- it's only going to match a single link per each of the teasers
- it works both with grids (e.g. labs experiments) and listings (e.g. collections)